src/js/CoreActionConsts.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  "AddCheck": "ADD_CHECK",
+  "Nextverse": "NEW_VERSE",
+  "PrevVerse": "NEW_VERSE",
+}
+
+/**
+Array that maps words to consts for use in actions
```

Comment:
You probably meant to say "Object that maps..". Don't have to fix now. Just include it in the next PR.